test/Test/Database/LSMTree/Internal/Vector/Growing.hs

```diff
@@ -0,0 +1,212 @@
+module Test.Database.LSMTree.Internal.Vector.Growing (tests) where
+
+import           Prelude hiding (head, length, tail)
+
+import           Control.Category ((>>>))
+import           Control.Monad.ST.Strict (ST, runST)
+import qualified Data.List as List (length)
+import           Data.Vector (Vector, toList)
+import           Database.LSMTree.Internal.Vector.Growing (GrowingVector,
+                     append, freeze, new)
+import           Test.QuickCheck (Arbitrary (arbitrary, shrink), Gen,
+                     NonNegative (NonNegative, getNonNegative), Positive
+                     (Positive), Property, Small (Small), Testable, chooseInt,
+                     shrinkList, shrinkMap, shrinkMapBy, sized, tabulate, (===))
+import           Test.Tasty (TestTree, testGroup)
+import           Test.Tasty.QuickCheck (testProperty)
+
+-- * Tests
+
+tests :: TestTree
+tests = testGroup "Test.Database.LSMTree.Internal.Vector.Growing" $
+        [
+            testProperty "Final vector is correct"
+                         prop_finalVectorIsCorrect
+        ]
```

Comment:
Tests tests are not being run yet. You should add include them in the `Main` module

Reply:
Oops, right. 🙂

Reply:
Fixed in 6f88b3f710bd5ae0b3996d8341435b74f9f7c37d.